docs/usage/delete.rst

```diff
@@ -7,6 +7,12 @@ Examples
     # delete a single backup archive:
     $ borg delete /path/to/repo::Monday
 
+    # delete all archives whose names begin with "T"
```

Comment:
is `whose` correct grammar?

can you use a more practical example prefix than `T`? (look at other places in the docs using `--prefix`)

Reply:
I believe "whose" is the correct usage